modules/users/manifests/user.pp

```diff
@@ -8,8 +8,12 @@
     String                  $shell      = '/bin/bash',
     Optional[Array[String]] $privileges = undef,
     Array[String]           $ssh_keys   = [],
+    Boolean                 $system     = false,
+    String                  $homedir    = '',
```

Comment:
personally not a fan of defaulting to an empty string i would instead do normally use `Optional[String[1] $foo = undef`  however in this case you can just default it to what you want as the default

```suggestion
    String                  $homedir    = "/home/${name}",,
```